src/banana/data/db.py

```diff
@@ -56,6 +56,7 @@ class Theory(Base):
     Qmc = Column(Float)
     Qmt = Column(Float)
     Qref = Column(Float)
+    Qedref = Column(Float)
```

Comment:
The name is a nice pun, but slightly confusing (or not 100% explicit). However, this is even not in our runcard, but completely internal to the benchmarking, so we could even keep it...

Reply:
Yes I agree but I used the old name that was in banana. I think we can change it once for all. All I can think of is `alphaem_ref` of `Qqedref`. If you have better suggestion I'm all ears

Reply:
Not really, I'm not so good at naming. 

- `Qqedref` is the more explicit version indeed, that's what I thought in the first place
- `QrefQED` (or every variation like `Qrefqed`) it solves the problem of the two `q`s in a line, and the two scales are alphabetically sorted closer (even if they are already side by side)
- `Qemref` it is a further alternative

Pick your favorite, and if the current one is the one you like the most, feel free to keep it :)